build.sbt

```diff
@@ -581,53 +577,13 @@ val allProjects = Seq(
   bloopgun213
 )
 
-val allProjectsToRelease = Seq[ProjectReference](
-  bloopShared,
-  backend,
-  frontend,
-  sbtBloop,
-  nativeBridge04,
-  jsBridge06,
-  jsBridge1,
-  sockets,
-  bloopgun,
-  bloopgunShaded,
-  bloopgun213,
-  bloopgunShaded213,
-  launcher,
-  launcherShaded,
-  launcher213,
-  launcherShaded213,
-  buildpressConfig,
-  buildpress
-)
-
 val allProjectReferences = allProjects.map(p => LocalProject(p.id))
 val bloop = project
   .in(file("."))
   .disablePlugins(ScriptedPlugin)
   .aggregate(allProjectReferences: _*)
   .settings(
-    releaseEarly := { () },
     (publish / skip) := true,
-    publishLocalAllModules := {
-      BuildDefaults
-        .publishLocalAllModules(allProjectsToRelease)
```

Comment:
It seems that `allProjectsToRelease` contained more project than `allProjectReferences`. In the diff there are all the shaded and the buildpress projects. Will they be released by `ci-release` or, is it okay not to release them?

Reply:
Ah good catch, yes I am missing a couple that I didn't realize weren't in there. I knew the shaded stuff wasn't in there, but I was hoping to get some thoughts on https://github.com/scalacenter/bloop/pull/1961 before finishing up this pr 😆. Apart from that it looks like I'm also missing the build press stuff. Just to compare here's what a `publishLocal` will give you:

```
 bloop-backend_2.12
 bloop-frontend_2.12
 bloop-js-bridge-0-6_2.12
 bloop-js-bridge-1_2.12
 bloop-launcher-core_2.12
 bloop-launcher-core_2.13
 bloop-launcher-test_2.12
 bloop-native-bridge-0-4_2.12
 bloop-shared_2.12
 bloopgun-core_2.12
 bloopgun-core_2.13
 sbt-bloop
 sockets
```
vs what was in the publish list before:

```
  bloopShared, // TODO do we actually need to publish this???
  backend,
  frontend,
  sbtBloop,
  nativeBridge04,
  jsBridge06,
  jsBridge1,
  sockets,
  bloopgun,
  bloopgunShaded,
  bloopgun213,
  bloopgunShaded213,
  launcher,
  launcherShaded,
  launcher213,
  launcherShaded213,
  buildpressConfig,
  buildpress
```
I'll make sure the others get added, but it'll be nice to get some thoughts on the shaded stuff first, because ideally we wouldn't have to include those here as well.

Reply:
Alright now that the shaded stuff is gone, the new publish list will look like this:

```
bloop-backend_2.12
bloop-frontend_2.12
bloop-js-bridge-0-6_2.12
bloop-js-bridge-1_2.12
bloop-launcher-core_2.12
bloop-launcher-core_2.13
bloop-launcher-test_2.12
bloop-native-bridge-0-4_2.12
bloop-shared_2.12
bloopgun-core_2.12
bloopgun-core_2.13
buildpress_2.12
buildpressconfig_2.12
sbt-bloop
sockets
```